dips/dip-14.md

```diff
@@ -0,0 +1,245 @@
+---
+dip: 14
+title: Multi-Agent Transactions
+author: Emma Zhong(@emmazzz), Tim Zakian, Sam Blackshear
+Status: Draft
+type: Standard
+created: 04/05/2021
+---
+
+
+# Summary
+
+This DIP describes a new scheme of user transaction: multi-agent transactions.
+
+# Abstract
+
+Currently in the Diem Framework, a single transaction acts on behalf of a single on-chain account. However, there is no mechanism for multiple on-chain accounts to agree on a single atomic transaction. This DIP presents a new scheme of transactions--multi-agent transactions--which act on behalf of multiple on-chain accounts. Multi-agent transactions leverage Move’s [_`signer`_](https://developers.diem.com/docs/move/move-signer/) type to allow essentially any arbitrary atomic actions in one transaction involving multiple on-chain accounts.
+
+## Terminology:
+
+* Primary signer: This is the account that the transaction is sent from. This account’s sequence number is incremented, and is the account that pays gas. There is precisely one of these for every transaction.
+* Secondary signer: This is any account that participates in a multi-signer/agent transaction that isn’t the primary sender. There can be 0-N of these. The 0 case is a normal transaction today. 
```

Comment:
I'm suggesting to avoid the "multi-signer" term here. This doc has consistently used the "multi-agent" term for the topic of this DIP, and there has often been confusion vs. the related idea of a "multi-sig" account.